snuba/environment.py

```diff
@@ -32,13 +29,4 @@ def setup_sentry() -> None:
     )
 
 
-clickhouse_rw = ClickhousePool(settings.CLICKHOUSE_HOST, settings.CLICKHOUSE_PORT)
-clickhouse_ro = ClickhousePool(
-    settings.CLICKHOUSE_HOST,
-    settings.CLICKHOUSE_PORT,
-    client_settings={"readonly": True},
-)
-
```

Comment:
There is a comment related with this here:
https://github.com/getsentry/snuba/pull/860#discussion_r402032120